src/elements/truss/elementTrussInternForce.m

```diff
@@ -70,18 +70,23 @@
     KTe   =   stress      * A / lini * Ge  ...
             + dstressdeps * A * lini * ( (b1 + b2)' * (b1 + b2) ) ;
 
-  elseif strcmp( hyperElasModel, '1DrotEngStrain') || strcmp( hyperElasModel, 'isotropicHardening')
+   elseif strcmp( hyperElasModel, '1DrotEngStrain')
```

Comment:
tenemos algo que discutir acá @JoaquinViera ... @Sergesto  plantea que en isotropic hardening solo se use logaritmica rotada, pero podemos dejar a elegir ambas medidas de def

Reply:
fixed in [317aaa3](https://github.com/ONSAS/ONSAS.m/pull/664/commits/317aaa38effb7e1ca520750fe64d08e92c7a129e)